config/gitlab.yml.example

```diff
@@ -80,6 +80,11 @@ production: &base
       wiki: true
       snippets: false
 
+      ## Default web hooks settings
```

Comment:
Ok, so for this PR the config needs to be moved to some web interface right? Can you say if you would like to see this merged in? If so I'll guide the rest of the PR.